framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.11 under development
 ------------------------
 
+- Enh #12798: Changed `yii\cache\Dependency::getHasChanged()` (to be deprecated in 2.1) to `yii\cache\Dependency::isChanged()` (dynasource)
```

Comment:
It is to be `removed` in 2.1. That's the important bit.

`"Replace yii\cache\Dependency::getHasChanged() by yii\cache\Dependency::isChanged()"` is a line for 2.1 UPGRADE.